<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>352</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QOpenGLWidget" name="openGLWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>50</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewScene"/>
   <addaction name="actionOpenScene"/>
   <addaction name="actionSaveScene"/>
   <addaction name="separator"/>
   <addaction name="actionReadme"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="hierarchyDock">
   <property name="minimumSize">
    <size>
     <width>169</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="hierarchy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>331</height>
      </rect>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="loadMeshButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Mesh</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboMesh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="inspectorDock">
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>38</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>370</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewScene"/>
    <addaction name="actionOpenScene"/>
    <addaction name="actionSaveScene"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReadme"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenScene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Folder.png</normaloff>:/icons/Resources/icons/Folder.png</iconset>
   </property>
   <property name="text">
    <string>Open scene</string>
   </property>
  </action>
  <action name="actionSaveScene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Disquette.png</normaloff>:/icons/Resources/icons/Disquette.png</iconset>
   </property>
   <property name="text">
    <string>Save scene</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNewScene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Document-new.png</normaloff>:/icons/Resources/icons/Document-new.png</iconset>
   </property>
   <property name="text">
    <string>New scene</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Resources/icons/Info.png</normaloff>:/icons/Resources/icons/Info.png</iconset>
   </property>
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
